packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.tsx

```diff
@@ -158,10 +160,18 @@ export class DetailsList extends BaseComponent<IDetailsListProps, IDetailsListSt
     }
   }
 
+  public getStartItemIndexInView(): number {
+    return this._list.current!.getStartItemIndexInView() || 0;
+  }
+
   public componentWillUnmount(): void {
     if (this._dragDropHelper) {
       this._dragDropHelper.dispose();
     }
+
+    if (this.props.onWillUnmount) {
```

Comment:
There isn't a need for this hook since the parent component can use its on `componentWillUnmount` hook to make calls on the `IDetailsList` instance.